README.md

```diff
@@ -31,7 +31,7 @@ azmi setblob --file ~/info.txt --blob $CONTAINER_URL/myhostname/info.txt --ident
 azmi getsecret --secret $SECRET_URL
 azmi getsecret --secret $SECRET_URL/$VERSION
 
-# fetch latest or specific version of a certificate from Azure Key Vault
+# fetch latest or specific version of a certificate (PEM or PFX format; certificate(s) and private key bundle) from Azure Key Vault
```

Comment:
Again, I think this explanation is too big. One will just download the certificate in the same format as it is uploaded, right?

Reply:
Okay, did a bit redo.